G/glslang/build_tarballs.jl

```diff
@@ -0,0 +1,34 @@
+using BinaryBuilder
+
+name = "glslang"
+version = v"8.13.3743"
+
+source = "https://github.com/KhronosGroup/glslang.git"
+commit = "bcf6a2430e99e8fc24f9f266e99316905e6d5134" # April 27th, 2020
+
+sources = [
+    GitSource(source, commit)
+]
+
+script_linux = raw"""
+cd glslang
+
+
+cmake -B build -S . -GNinja ${CMAKE_FLAGS[@]}
```

Comment:
Where is `CMAKE_FLAGS` coming from?  See https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#CMake-builds for the arguments to pass to Cmake, all of `-DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}` are fundamental, `-DCMAKE_BUILD_TYPE=Release` is optional but strongly recommended

Reply:
Whoops, I must have deleted those by accident.. I took the BinaryBuilder.jl examples and I recall having tested this build_tarballs with the mentioned flags.